Sources/AdminPanelProvider/Controllers/AdminPanelUserController.swift

```diff
@@ -92,7 +93,8 @@ public final class AdminPanelUserController {
             {
                 var context: ViewData = try [
                     "user": user.makeViewData(),
-                    "name": .string(name)
+                    "name": .string(name),
+                    "url": .string(panelConfig.baseUrl)
```

Comment:
As discussed IRL, I think we should append `/admin` in the Leaf template to be consistent with the reset flow.